lib/Localization/LocalizationFormat.cpp

```diff
@@ -85,9 +94,22 @@ bool SerializedLocalizationWriter::emit(llvm::StringRef filePath) {
   return OS.has_error();
 }
 
+llvm::StringRef
+LocalizationProducer::getMessageOr(swift::DiagID id,
```

Comment:
I forward declared `getMessageOr` to be able to do faster incremental builds. When modifying a `.h` file and if it's included in multiple places, it takes much longer to build than modifying a `.cpp` file

Reply:
Sounds good!